source/guide_kirby.rst

```diff
@@ -0,0 +1,161 @@
+.. sectionauthor:: Daniel Kratz <uberlab@danielkratz.com>
+.. highlight:: console
+
+.. sidebar:: Logo
+  
+  .. image:: _static/images/kirby.svg 
+      :align: center
+
+#########
+Kirby
+#########
+
+Kirby_ is a modern, fast, flexible, file-based (no database) content management system written in PHP. 
+
+The idea behind Kirby is to provide a tool for developing websites or applications that is both easy to use for developers and at the same time gives you all the freedom you want to structure your content and render your output.
+
+Kirby was released for the first time in 2009 and is actively developed by the Bastian Allgeier GmbH. Although the code is open source, a license is required for use in production. 
```

Comment:
No, you need a license for every single website as stated in the [license overview](https://getkirby.com/buy) (starting with the first). That's also what I'm reading in the cited license terms. Do you think it's enough if I include the additional warning and the disclaimer under Configuration where I set the license? That would maybe mess up the intro otherwise because of the length. In the intro I could change "a license is required" to "a per website license is required" to clarify that.